models/Tag.js

```diff
@@ -1,13 +1,19 @@
 var keystone = require('@twreporter/keystone');
 var transform = require('model-transform');
+const { Types } = keystone.Field
 
 var Tag = new keystone.List('Tag', {
 	autokey: { from: 'name', path: 'key', unique: true },
 });
 
 Tag.add({
 	name: { label: '標籤名稱', type: String, required: true },
-	latest_order: { label: '最新(順序)', type: Number, noedit: true },
+	latest_order: {
+          type: Types.Tag,
```

Comment:
changing the type would break the logics depending on it, like numeric operations in this case(sorting, value comparison...etc), we may need to be careful about modifying something fundamental

Reply:
In [keystone PR](https://github.com/twreporter/keystone/pull/233), `Number` type is inherited as `Tag` native type.
I think it could fullfill current needs (Latest page cannot store `latest_order` as number, and Tag can show it) for now.
In the future, it would great if we implement [`virtual` type](https://keystonejs.com/docs/fields/virtual) as a more general solution.

Reply:
I think it's good for us to have a new tag type, however semantically people seeing `latest_order` would expect it's an numeric order, and this `tag` type would more be an attribute to the parent `Tag` rather than the child field `latest_order`(i.e. Tag `2016總統大選` is marked as `最新`), so I would suggest we append the new tag label like:

<img width="619" alt="Screen Shot 2023-02-13 at 3 20 36 PM" src="https://user-images.githubusercontent.com/25971696/218396536-06cc54c9-5f58-420d-abe6-287e585b0643.png">

And I would also hope we keep the value of `latest_order` visible in case we need trouble shooting, thanks.

Reply:
After discussion with designer, the latest tag & latest_order number has duplicate function for user, thus we decide to keep latest_order number only.
Change applied in commit [49797a5](https://github.com/twreporter/plate/pull/347/commits/49797a55148b050ea0906a94f0b6bd1eace0eaa1), please have a look.